bundler/src/main.rs

```diff
@@ -76,12 +77,33 @@ where
     }
 }
 
+/// Wrapper to ensure that globs passed via the CLI are valid file globs
+#[derive(Debug, Clone)]
+struct StaticDataGlob(String);
+
+impl FromStr for StaticDataGlob {
+    type Err = PatternError;
+
+    fn from_str(value: &str) -> Result<Self, Self::Err> {
+        _ = Pattern::new(value)?;
+        Ok(Self(value.into()))
+    }
+}
+
+impl AsRef<str> for StaticDataGlob {
+    fn as_ref(&self) -> &str {
+        &self.0
+    }
+}
```

Comment:
Can probably use `derive_more::AsRef` here

Reply:
Didn't notice `derive_more` was already included

Reply:
I made sure to check this time :sweat_smile: